source/codegen/metadata/nifgen/functions.py

```diff
@@ -1471,6 +1537,30 @@
         ],
         'returns':'ViStatus'
     },
+    'ErrorHandler':{
+        'parameters':[
+            {
+                'name':'vi',
+                'direction':'in',
+                'type':'ViSession'
+            },
+            {
+                'name':'errorCode',
+                'direction':'in',
+                'type':'ViStatus'
+            },
+            {
+                'name':'errorMessage',
+                'direction':'out',
+                'type':'ViChar[]',
+                'size':{
+                    'mechanism':'fixed',
+                    'value':256
```

Comment:
You were right about using fixed mechanism and value 256 for this, even in the niFgen.h they have used the same value : 
![image](https://user-images.githubusercontent.com/47028785/120787914-7c177d80-c54d-11eb-8f3d-41818b1dbaaf.png)

Reply:
Oh ok. Thanks Akshata